src/backend/DocDownloaderAPI/src/main/resources/logback-splunk.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+    <include resource="org/springframework/boot/logging/logback/defaults.xml"/>
+    ​
+    <!-- You can override this to have a custom pattern -->
+    <property name="CONSOLE_LOG_PATTERN"
```

Comment:
In this case we don;t need the correctionId. What we do need though is %X{transferid} in it's place.